### ISSUE TITLE
[Error] NOAA images not working but METEOR images ar

### ISSUE THREAD

**Issue by aero-oli** [jekhokie/raspberry-noaa-v2#517]
Hello, as you can see in the attached images, the METEOR ones are pretty good, but the NOAA ones all look like the attached image. no idea why this is. 
RTL-SDR V3
set up using the image.
![NOAA-18-20220315-105253-pristine](https://user-images.githubusercontent.com/73533343/158671117-480ac861-d9a6-46fb-a0fe-85bc54933ac0.jpeg)
![METEOR-M-2-20220315-173230-122-rectified](https://user-images.githubusercontent.com/73533343/158671136-dabb673a-683d-4114-b732-c3367fb8f9d0.jpeg)
[output.log](https://github.com/jekhokie/raspberry-noaa-v2/files/8269331/output.log)

**Comment by betolley** [comment]
Take a look at this.  I don't have a answer for you.  https://github.com/jekhokie/raspberry-noaa-v2/issues/515

I was using a orange pi one with similar results.  When you listened to the audio it played too fast.  It caused NOAA
 issues when converting audio.  But METEOR was ok.    I ended up moving to a intel based Chromebox that I loaded with Armbian x86 and setup.  This is my most recent pass over with same hardware otherwise
![NOAA-18-20220504-120740-MCIR](https://user-images.githubusercontent.com/12071323/166851544-9a2914d5-0a73-4f80-a515-b6ef592eac8b.jpg)

.
